### ISSUE TITLE
Failed to activate the atom-sassc-live package

### ISSUE THREAD

**Issue by steweir** [adekbadek/atom-sassc-live#4]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.2 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.10.5
**Thrown From**: [atom-sassc-live](https://github.com/adekbadek/atom-sassc-live) package 1.0.8

### Stack Trace

Failed to activate the atom-sassc-live package

```
At path is not defined

ReferenceError: path is not defined
    at isSassFile (/packages/atom-sassc-live/lib/atom-sassc-live.coffee:46:3)
    at /packages/atom-sassc-live/lib/atom-sassc-live.coffee:154:10
    at Workspace.observeTextEditors (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:717:15)
    at Object.activate (/packages/atom-sassc-live/lib/atom-sassc-live.coffee:152:20)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:253:25)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:968:31
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:246:26)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1338:34)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app/src/application-delegate.js:334:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -0:16.1.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always)
     -0:03.6.0 atom-sassc-live:toggle (input.hidden-input)
```

### Non-Core Packages

```
apex-syntax 0.1.0 
apex-ui 1.0.0 
atom-beautify 0.30.5 
atom-browser 0.53.0 
atom-commander 0.10.1 
atom-img 1.0.3 
atom-material-syntax 1.0.6 
atom-runner 2.7.1 
atom-sassc-live 1.0.8 
atom-watcher 1.1.7 
browser-plus 0.0.98 
build 0.68.0 
busy 0.7.0 
busy-signal 1.4.3 
color-picker 2.2.5 
colorpicker 0.3.0 
command-toolbar 1.0.12 
dev-tools-themes 0.1.3 
file-icons 2.1.11 
file-watcher 1.1.0 
filetype-color 0.1.4 
flex-tool-bar 0.12.0 
intentions 1.1.5 
intentions-colorpicker 1.0.1 
linter 2.2.0 
linter-ui-default 1.6.7 
livesave 0.0.0 
livestyle-atom 0.2.9 
lucid-tabs 1.1.1 
material-ui 1.0.10 
minimap 4.29.6 
open-recent 5.0.0 
open-terminal-here 2.3.1 
pane-browser 1.6.0 
pigments 0.40.2 
pinned-tabs 1.3.0 
project-dropdown 0.13.0 
project-ring 0.40.0 
psd-extract 0.1.1 
recent-projects 0.13.1 
toggle-gutter 0.3.3 
tool-bar 1.1.0 
tool-bar-almighty 0.8.2 
web-view 3.0.0 
```

**Comment by adekbadek** [comment]
Hi @steweir! I was unable to reproduce your issue, but have published a possible fix in v1.0.10. Please update and let me know if it's resolved.

**Comment by adekbadek** [comment]
closing due to lack of feedback
